problems/[230619] Top K Frequent Words.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/top-k-frequent-words
+ *
+ * @param {string[]} words
+ * @param {number} k
+ * @return {string[]}
+ */
+const topKFrequent = function(words, k) {
+  const map = {};
+
+  words.forEach((word) => {
+    if (map[word]) return map[word] += 1;
+      map[word] = 1;
```

Comment:
```suggestion
    imap[word] ? map[word] += 1 : map[word] = 1;
      
```

삼항 연산자로 더 깔끔하게 쓸 수 있을 것 같습니다!